scripts/upload_test_files_to_azure.sh

```diff
@@ -5,10 +5,16 @@
 
 # Create container
 az storage container create -n testing-private --account-name $AZURE_STORAGE_ACCOUNT
-az storage container create -n testing-public --account-name $AZURE_STORAGE_ACCOUNT --public-access container
+az storage container create -n testing-public  --account-name $AZURE_STORAGE_ACCOUNT --public-access container
```

Comment:
I couldn't really get this line to work, did you? somehow the `--public-access container` prevents creation.

This does work for the non-hierarchical storage account

Reply:
FYI: I change the script to let user known that we expect DFS option.
Can you confirm that the public access is enable on your storage ?
![image](https://github.com/duckdb/duckdb_azure/assets/3530021/63e32d7b-3e4c-44ec-8463-58aee9c41452)

Reply:
yeep that'll do it, thanks :)